test/system/test_cvp_api.py

```diff
@@ -73,54 +73,61 @@ class TestCvpClient(DutSystemTest):
     '''
     # pylint: disable=too-many-public-methods
     # pylint: disable=invalid-name
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
         ''' Instantiate the CvpClient class and connect to the CVP node.
             Log messages to the /tmp/TestCvpClient.log
         '''
-        super(TestCvpClient, self).setUp()
-        self.clnt = CvpClient(filename='/tmp/TestCvpClient.log')
-        self.assertIsNotNone(self.clnt)
-        self.assertIsNone(self.clnt.last_used_node)
-        dut = self.duts[0]
+        super(TestCvpClient, cls).setUpClass()
+        cls.clnt = CvpClient(filename='/tmp/TestCvpClient.log')
+        assert cls.clnt is not None
+        assert cls.clnt.last_used_node is None
+        dut = cls.duts[0]
         cert = False
         if 'cert' in dut:
             cert = dut['cert']
```

Comment:
Add checks for api_token and is_cvaas parameters similar to how cert is handled. This will prevent dictionary key errors if no value is provided in cvp_nodes.yaml

Reply:
Added all checks as per your suggestion